src/reporting/market.se

```diff
@@ -44,10 +45,13 @@ data tentativeWinningPayoutDistributionHash
 data finalPayoutDistributionHash
 data automatedReporterPayoutDistributionHash
 data automatedReporterDisputeBondHolder
+data automatedReporterDisputeBondToken
```

Comment:
I'm not sure I understand this part.  Are you saying get rid of automatedReporterDisputeBondToken, limitedReportersDisputeBondToken and allReportersDisputeBondToken?  If so, where would I have the factory create them?